features/performance/manifests/templates/00-machine-config-pool-worker-rt.yaml.in

```diff
@@ -0,0 +1,14 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-rt
+  labels:
+    worker-rt: ""
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,worker-rt]}
```

Comment:
Both worker and worker-rt?
Doesn't this conflict with the default worker pool?

Reply:
as far as I saw no, and we can not update `KubeletConfig` without providing a default config file, see - https://bugzilla.redhat.com/show_bug.cgi?id=1772490

Reply:
I see, thx :+1: